pyramid_rpc/jsonrpc.py

```diff
@@ -267,7 +267,7 @@ def text(self):
     phash = text
 
     def __call__(self, context, request):
-        return getattr(request, 'rpc_method') == self.method
+        return hasattr(request, 'rpc_method') and getattr(request, 'rpc_method') == self.method
```

Comment:
`getattr(request, 'rpc_method', None)` would be the best fix for this. I forgot that `getattr` does not default to `None`.